auth/build.gradle

```diff
@@ -50,6 +50,9 @@ dependencies {
     // library when updating com.android.support libraries:
     compile "com.android.support:cardview-v7:$support_library_version"
 
+    compile 'com.squareup.retrofit2:retrofit:2.1.0'
+    compile 'com.squareup.retrofit2:converter-gson:2.1.0'
```

Comment:
No worries about these, Twitter's already includes them and I'm just upgrading it for them:
```
+--- com.twitter.sdk.android:twitter:2.3.0
|    +--- com.twitter.sdk.android:tweet-composer:2.3.0
|    |    +--- com.twitter:twitter-text:1.13.0
|    |    +--- io.fabric.sdk.android:fabric:1.3.14
|    |    +--- com.twitter.sdk.android:twitter-core:2.3.0
|    |    |    +--- com.google.code.gson:gson:2.6.1 -> 2.7
|    |    |    +--- com.squareup.retrofit2:converter-gson:2.0.2 -> 2.1.0 <<<<<<<<<<<<
|    |    |    |    +--- com.squareup.retrofit2:retrofit:2.1.0
|    |    |    |    |    \--- com.squareup.okhttp3:okhttp:3.3.0
|    |    |    |    |         \--- com.squareup.okio:okio:1.8.0
|    |    |    |    \--- com.google.code.gson:gson:2.7
|    |    |    +--- com.squareup.okhttp3:okhttp:3.2.0 -> 3.3.0 (*)
|    |    |    +--- io.fabric.sdk.android:fabric:1.3.14
|    |    |    \--- com.squareup.retrofit2:retrofit:2.0.2 -> 2.1.0 (*)
|    |    \--- com.squareup.picasso:picasso:2.5.2
|    +--- io.fabric.sdk.android:fabric:1.3.14
|    +--- com.twitter.sdk.android:twitter-core:2.3.0 (*)
|    \--- com.twitter.sdk.android:tweet-ui:2.3.0
|         +--- io.fabric.sdk.android:fabric:1.3.14
|         +--- com.twitter.sdk.android:twitter-core:2.3.0 (*)
|         +--- com.android.support:support-v4:23.1.1 -> 25.0.1 (*)
|         \--- com.squareup.picasso:picasso:2.5.2
```